src/routes/MetaDetails/StreamsList/Stream/styles.less

```diff
@@ -105,6 +113,33 @@
     }
 }
 
+.context-menu-container {
```

Comment:
This should be inside `.stream-container` otherwise it applies to every `.context-menu-container` elements